runtime/doc/change.txt

```diff
@@ -196,6 +196,7 @@ gR			Enter Virtual Replace mode: Each character you type
 
 							*v_r*
 {Visual}r{char}		Replace all selected characters by {char}.
+			CTRL-C will be inserted literally.
```

Comment:
How is this useful? Can't ctrl-v ctrl-c be used for that already? ctrl-c should always cancel the mode.

Reply:
But it's broken

Reply:
https://github.com/vim/vim/pull/13112#issuecomment-1727283535

This is fine for now, but ideally I would think we want ctrl-c to cancel the mode (except after ctrl-v).

Reply:
For the record, ctrl-c can bring you to insert mode. For instance, when you start in insert mode and type `<c-o> v r <c-c>`. There are also variants which steps through select mode such as `<c-o> v <c-g> <c-o> r <c-c>` or command line mode `<c-o> : <c-c>`.